src/main/resources/templates/fridge.html

```diff
@@ -41,29 +41,29 @@ <h1> Zawartość lodówki </h1>
       <th> Nazwa</th>
       <th> Ilość</th>
       <th> Jednostka miary</th>
-      <th> Data ważności</th>
+      <th> Data ważności
+      <th> Opcje</th>
     </tr>
     </thead>
-    <tbody>
 
+    <tbody>
     <tr th:each="product: ${productsinfridge.getProductsInFridge()}">
       <td th:text="${product.productName}"></td>
       <td th:text="${product.amount}"></td>
       <td th:text="${product.unit.value}"></td>
       <td th:text="${product.expirationDate}"></td>
-    </tr>
+      <td>
+<!--        <a th:href="@{/fridge/product/{productId}(productId=${product.productId})}"-->
+<!--           class="btn btn-primary">Edytuj</a>-->
```

Comment:
Czemu zakomentowane?

Reply:
Bo to nie ta funkcjonalność (inny task) a guzik na przyszlosc sie przyda.